source/css/_layout.grids.scss

```diff
@@ -85,6 +85,7 @@
   > * {
     padding-left: $space-mobile;
     padding-right: $space-mobile;
+    box-sizing: border-box;
```

Comment:
Was this not inheriting from the [reset css](https://github.com/adventistchurch/alps/blob/ea5d8b5c931cee53161917b900057d9f53f5b4d2/source/css/_generic.reset.scss#L6)?